### ISSUE TITLE
`dev` how do I get rid of vertical lines 2 and 4? and their labels?

### ISSUE THREAD

**Issue by edwardtfn** [Blackymas/NSPanel_HA_Blueprint#1877]
Question,
how do I get rid of vertical lines 2 and 4? and their labels?
![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/17070805/2bf3df9b-a234-4c7e-982b-311c6b686eed)
also only label 1 is working,
for the others I did not assign a value only label and image, could that be the reason?
My utilities values are
```
      qrcode_enabled: false
      utilities_enabled: true
      utilities_page01_home_value1: sensor.energy_consumed_tariff_1
      utilities_page01_home_value2: sensor.energy_consumed_tariff_2
      utilities_page01_group04_line_inverted: false
      utilities_page01_group01_value1: sensor.grid_fossil_fuel_percentage
      utilities_page01_group01_label: Low-carbon
      utilities_page01_group01_icon: mdi:leaf
      utilities_page01_group01_color:
      - 15
      - 157
      - 88
      utilities_page01_group02_icon: ''
      utilities_page01_group03_icon: mdi:solar-power
      utilities_page01_group03_label: Solar
      utilities_page01_group06_icon: mdi:water
      utilities_page01_group06_label: Water
      utilities_page01_group05_label: Gas
      utilities_page01_group05_icon: mdi:fire
```

_Originally posted by @bkbartk in https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/168#issuecomment-1987346478_

**Comment by edwardtfn** [comment]
Could you please try again with v4.3.0b2?

**Comment by bkbartk** [comment]
the vertical lines are gone as expected,
but there are no labels at all
![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/17070805/9f5be405-754d-4afe-b489-169212c86982)
just to be sure I resaved the blueprint but it didn't help
```
      qrcode_enabled: false
      utilities_enabled: true
      utilities_page01_home_value1: sensor.energy_consumed_tariff_1
      utilities_page01_home_value2: sensor.energy_consumed_tariff_2
      utilities_page01_group04_line_inverted: false
      utilities_page01_group01_value1: sensor.grid_fossil_fuel_percentage
      utilities_page01_group01_label: Low-carbon
      utilities_page01_group01_icon: mdi:leaf
      utilities_page01_group01_color:
      - 15
      - 157
      - 88
      utilities_page01_group02_icon: ''
      utilities_page01_group03_icon: mdi:solar-power
      utilities_page01_group03_label: Solar
      utilities_page01_group06_icon: mdi:water
      utilities_page01_group06_label: Water
      utilities_page01_group05_label: Gas
      utilities_page01_group05_icon: mdi:fire
```

**Comment by bkbartk** [comment]
I tried some real values,
and it's hare to see, next to the grid value
but it's saying None
![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/17070805/5680376c-e738-4bcd-b54b-4ab86294f4bc)
config:
```
      qrcode_enabled: false
      utilities_enabled: true
      utilities_page01_home_value1: sensor.energy_consumed_tariff_1
      utilities_page01_home_value2: sensor.energy_consumed_tariff_2
      utilities_page01_group04_line_inverted: false
      utilities_page01_group01_value1: sensor.grid_fossil_fuel_percentage
      utilities_page01_group01_label: Low-carbon
      utilities_page01_group01_icon: mdi:leaf
      utilities_page01_group01_color:
      - 15
      - 157
      - 88
      utilities_page01_group02_icon: ''
      utilities_page01_group03_icon: mdi:solar-power
      utilities_page01_group03_label: Solar
      utilities_page01_group06_icon: mdi:water
      utilities_page01_group06_label: Water
      utilities_page01_group05_label: Gas
      utilities_page01_group05_icon: mdi:fire
      home_value04: sensor.temp_outside_temperature
      home_value04_icon: mdi:sun-thermometer-outline
      utilities_page01_group03_value1: sensor.solaredge_ac_energy_kwh_today
      utilities_page01_group05_value1: sensor.gas_consumed_today
      utilities_page01_group06_value1: sensor.watermeter_today
      utilities_page01_grid_value2: sensor.energy_consumed_today
```
I treid both `utilities_page01_grid_value1` and `utilities_page01_grid_value2`
but is always says 0.418None

My sensor value does not contains None
![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/17070805/1ea2dc38-5333-45f4-979b-4d83030720c7)

**Comment by edwardtfn** [comment]
I believe this is fixed with the latest beta. Could you please update your system and let me know the results?

**Comment by bkbartk** [comment]
all the above is fixed on the release 4.3
